ui/src/VoxettaApp.js

```diff
@@ -30,3 +26,36 @@ export class VoxettaApp extends LitElement {
 }
 
 customElements.define('vox-app', VoxettaApp);
+
+
+navigator.mediaDevices.getUserMedia({ audio: true, video: false })
+    .then(function(stream) {
+
+        const recordButton = document.getElementById("recordButton");
+        const audioSave = document.getElementById("utterance");
```

Comment:
How is audioSave used? only the style is updated

Reply:
At the very bottom the src for audioSave is changed to the recordingURL with `audioSave.src = recordingURL;` which gives the audio file to the audio tag so it can play it back